app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -252,8 +254,11 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
             else -> {}
         }
 
-        binding.searchEnginesShortcutButton.setOnClickListener {
-            interactor.onSearchShortcutsButtonClicked()
+        binding.searchEnginesShortcutButton.apply {
```

Comment:
nit: just for consistency, for `qrScanButton` below we have the `setOnClickListener` and `increaseTapArea` on separate lines. Can we have the same here?

Reply:
fixed